style.css

```diff
@@ -504,7 +744,371 @@ ul {
   font-size: 17px;
   font-weight: 400;
   line-height: 20px;
-  letter-spacing: 0.01em;
   margin-bottom: 20px;
   color: #172b4d;
 }
+
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+  }
+
+  .logo {
+    font-size: 18px;
+    font-weight: 850;
+    line-height: 20px;
+    background-color: transparent;
+    margin-left: 100px;
+  }
+
+  .app-header {
+    display: flex;
+  }
+
+  .humbourger {
+    display: none;
+  }
+
+  .menu {
+    display: flex !important;
+    justify-content: flex-end;
+    align-items: center;
+    margin-right: 70px;
+    gap: 15%;
+  }
+
+  .menu-item a {
+    display: flex;
+    color: #344563;
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  .header-ambony {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+    background-color: #fff;
+    background-image: url(image/Header\ bg.svg);
+    background-size: cover;
+    top: 2.875rem;
+    border-bottom-left-radius: 60px;
+    height: 44.88rem;
+  }
+
+  .header-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+  }
+
+  .narson {
+    width: 100%;
+  }
+
+  .detail {
+    height: 98px;
+    width: 920px;
+    left: 0;
+    top: 134px;
+  }
+
+  .connection {
+    margin-top: -1px;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #7f8cff;
+  }
+
+  .work {
+    display: grid;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    margin: 10px 0 0 28px;
+    height: auto;
+    padding-right: 10px;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    padding: 24px;
+    gap: 46px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    margin-top: 100px;
+    height: 60vh;
+    width: 100%;
+    border-radius: 24px;
+  }
```

Comment:
- [ ] Kindly ensure that the project card is aligned correctly in the webpage layout, currently, it sticks to the right side, and should have the same amount of spacing on both sides 👍🏼 
- [ ] The spacing between the content should be respected as well from the Figma Design system as shown in the below screenshots 💪🏼 

| Your Design | Figma Design |
|---|---|
| ![image](https://user-images.githubusercontent.com/68030297/230695134-dc0b5b0a-ac0d-42b0-8140-8f10a8c07cec.png) | ![image](https://user-images.githubusercontent.com/68030297/230695171-d95438b4-d975-430a-af33-cea0ee81e035.png) |